testsuite/shared/src/main/java/org/jboss/as/test/integration/auditlog/AuditLogToTLSSyslogSetup.java

```diff
@@ -105,6 +198,7 @@ protected SyslogServerConfigIF getSyslogConfig() {
     public void setup(ManagementClient managementClient) throws Exception {
         FileUtils.deleteDirectory(WORK_DIR);
         WORK_DIR.mkdirs();
+        beforeTest();
```

Comment:
Are the three ```createTestResource()``` calls below still needed here? Looks like these are being created from ```beforeTest()``` now.

Reply:
They are still needed because they copy the created files to audit-workdir, which is best done using those calls because of where the directory is located